README.md

```diff
@@ -50,6 +50,9 @@ Assuming `wttrbar` is in your path, it can be used like:
 },
 ```
 
+## Icons dependencies
+- ttf-jetbrains-mono-nerd (Optional)
```

Comment:
I'd suggest removing this section. Under the already existing "Icons" section we can mentoin that if using the `--nerd-fonts` option you should make sure you have on installed. jetbrains-mono-nerd is just one out of many. and I mean, many: https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts